### ISSUE TITLE
Еще одна идея

### ISSUE THREAD

**Issue by Kamui-Solo** [kosmodrey/uNote#19]
Если возможно, то хотелось бы реализацию html тэгов иди bbcode, Например написал 
<a href="http://www.gearbest.com/headsets/pp_131206.html">syllable g600</a> затем переоткрыл заметки, а там уже только текст "syllable g600" который является ссылкой.
Или вообще чтобы наворочено было, панельку кнопками, как на форумах в поле ответа, но только с основными кнопками.

**Comment by kosmodrey** [comment]
Писать какие-либо заметки в виде HTML или BBCode - не самая лучшая идея, там возникают уже другие проблемы, в том числе и с безопасностью (не пройдёт проверки на addons.mozilla.org), да и не хочется из простых заметок делать комбайн с дополнительными кнопками.

Но есть решение - так называемые [WYSIWYG](http://en.wikipedia.org/wiki/WYSIWYG) редакторы, которые позволяют создавать ссылки, картинки и красиво оформлять текст, но я пока нашёл хорошей реализации. Следующая версия будет кругленькая - 1.0.0, думаю это будет неплохое новшество для заметок, просто надо подумать как правильно реализовать эти возможности, оставив простату и лёгкость дополнения.

**Comment by Kamui-Solo** [comment]
Да, WYSIWYG тоже отлично было бы. С нетерпением буду ждать новую версию.
